README.md

```diff
@@ -46,7 +46,7 @@ We welcome contributions to the code base. If you are interested in becoming a c
 - [Code contribution guidelines](/.github/CONTRIBUTING.md#Code-Contribution)
 
 
-There is a [specific channel called develop](https://discord.nos.io/) on Discord to discuss development.
+There is a [specific channel called develop](https://discord.gg/CXZb3BS) on Discord to discuss development.
```

Comment:
How come we're changing this link?  The existing one seems to work fine and matches the Discord linked from https://nos.io.

Reply:
Yep, however this one is a *direct* invite to develop, which seemed appropriate regarding the context. If you prefer the general discord link, I'll change it back

Reply:
This is fine, thanks for explaining